### ISSUE TITLE
The settings options aren't displayed properly.

### ISSUE THREAD

**Issue by obakha** [PranshulGG/WeatherMaster#109]
## Bug Description
When I open the settings tab, the text isn't displayed properly, also it's not clickable.

### Steps to Reproduce
1. Open the app.
2. Go to 'Settings'
3. See options displayed as regular text.
4. Click on any option/text.
4. Nothing is triggered.

### Expected Behavior
Open settings, options are well displayed , and clickable.

### Actual Behavior
Text is displayed as regular text, and it's not clickable.

### Screenshots
![](https://i.ibb.co/sWmwwJ8/eec88c40-3683-4582-996c-5aad2aafab0d.jpg)

### Environment
- OS: [Android]
- App version: [1.7.6 W]
- Device: [Samsung A04]
- Browser (if applicable).

### Additional Context
Add any other context or logs about the problem here.

**Comment by PranshulGG** [comment]
This is new, never seen this issue before. Looks like the material library has a loading problem or something, not really sure. Did it just start happening now, or was it working before?

**Comment by obakha** [comment]
I just downloaded the app today, so this was my first time facing this bug.

I'm on Android 10.

**Comment by obakha** [comment]
@PranshulGG Hi there, I have installed the new update, but the same issue still occurs.

**Comment by PranshulGG** [comment]
Well, I might need to dig into this issue. Not really sure what’s causing it.

**Comment by PranshulGG** [comment]
Nope, the app doesn't require any Google services. The fact that these issues are only happening for you and I can't reproduce them makes it hard to figure out what's causing it. Would you be down to test it out so we can try to solve it?

**Comment by obakha** [comment]
Sure, I’m happy to help test things out. Just let me know what steps I need to follow, and I’ll provide feedback.

Thanks for looking into this!

**Comment by PranshulGG** [comment]
Do you use any socials like Discord or something? You can join our server if you want!

**Comment by PranshulGG** [comment]
> This is strange as I'm not able to see this issue either. Are you able to test the app in any other phone?

From what I can see, the material library bundle isn’t getting rendered for some reason. But on the devices I tested, even on Android 7, it works fine.

**Comment by PranshulGG** [comment]
Maybe fixed!

**Comment by obakha** [comment]
I wished it was the case, but still the same thing. I really want to say that maybe the problem is my phone, but I've been using another weather app "Breezy Weather" and never had this issue with it. 

Don't worry I will still use yours, hopefully someday it'll get fixed.

**Comment by obakha** [comment]
What about adding a troubleshoot/debug option (that will test if everything is working alright.)

**Comment by obakha** [comment]
I really appreciate your effort. But I guess there could be a chance that it's just my phone being funny, but I hope this console log thing would finally solve the mystery.

**Comment by PranshulGG** [comment]
Well, I did some research, and it could be the issue. Some older devices might have trouble with certain JavaScript features. Make sure your WebView is the latest version.

**Comment by obakha** [comment]
You're a genius! I just noticed there's a new version available for my webview, I download it, and now everything is displayed as expected.

**Comment by obakha** [comment]
Same bro, I'm really glad!!

Can I ask you something? 
I'm trying to figure out how to setup Accweather API key in the app, I pasted the API key in the proper box, it said API was verified. Then when I try to open my city page, it says API error, I wonder if I missed something when generating the API key from accuweather site. 

Here's my configuration: 
![config](https://i.ibb.co/0tv2cSK/22bc2ac6-c1d0-4af7-8c03-8a0aba1ff6cb.jpg)

**Comment by PranshulGG** [comment]
The new version 1.7.9 changes how locations are saved, and I forgot to update that in the AccuWeather release version. Since it can’t verify the API key, you might have to wait until the next release. Would you mind opening a new issue about it?
